### DISCUSSION TITLE
Plus de gemmes !

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#2386] [Nouvelles idées]
Bonsoir, j'aimerais vous soumettre une idée pour avoir plus de gemmes. À l'heure actuelle, les gemmes sont une ressources très rare dans le jeu mais très utile pour les acharnés de île. Une fois l'émission de campagne terminée, il est difficile pour les grands joueurs ou même les autres d'en gagner . Et au vu de  c'est dernier jours je remarque que les missions quotidiennes ne rapporte pas grand-chose. Pour cela, nous avons eu l'idée avec la team Océan qu'une fois qu'un animal soit dressé et ça pourrait être en fonction de sa rareté que celui-ci puisse trouver des gemmes pendant des mini Event.

**Comment by Ravenclaw-l** [comment]
J'aimerais également proposer une amélioration concernant les 💎 
Il arrive régulièrement que les missions quotidiennes génèrent un nombre limité de gemmes au cours de certaines semaines. Les mêmes missions de 1 ou 2 💎  se répètent inlassablement.

Par exemple, lors de la semaine du 22 au 28 janvier, le total des gemmes obtenues s'est limité à 12. 
Ça ne couvre même pas le coût du 2ème voyage vers l'île, qui est de 15 💎 

Récap de la semaine : 
22 janvier : faire 3 rapports = 1 💎 
23 janvier : boire 1 potion sans effet = 2 💎 
24 janvier : rencontrer 2 joueurs différents = 1 💎 
25 janvier : récolter 250 💰 = 2 💎 
26 janvier : récolter 250 💰 = 2 💎 
27 janvier : boire 1 potion sans effet = 2 💎 
28 janvier : gagner 10 points de ❤️  = 2 💎 

Serait-il possible que toutes les missions proposées au cours d'une semaine offrent un minimum de 25 💎 en récompense ?
Je trouve que 25 💎 n'est pas excessif compte tenu du fait qu'il faut 40 💎 pour faire 3 voyages sur l'île.

**Reply by Voisin1234** [reply]
Je valide complètement 👍

**Comment by Procion06** [comment]
j'aime bien vos 2 propositions
un pet 4 ou 5* dressé qui trouve des gemmes (entre 1 et 3 par ex) dans les mini events ça a du sens, comme pour les objets, sachant que ça reste un mini event plutôt rare
concernant les missions quotidiennes, étant données qu'elles sont aléatoires, le plus simple serait juste d'augmenter les récompenses en gemmes
une récompense d'une ou 2 gemmes c'est vraiment pas fifou, surtout que ton récap se base sur un joueur qui valide toutes ses missions quotidiennes, ce qui est loin d'être le cas de la majorité
